examples/text-generation/text-generation-pipeline/run_pipeline_langchain.py

```diff
@@ -0,0 +1,86 @@
+import argparse
```

Comment:
I don't see any license in scripts within current folder (```examples/text-generation/text-generation-pipeline```), but I see license present in scripts in parent folder ```examples/text-generation```. Please clarify if some license needed

Reply:
You can add the apache-2.0 license (see an example [here](https://github.com/huggingface/optimum-habana/blob/main/optimum/habana/version.py)) and copyrights for Hugging Face and Intel.

Reply:
thanks, added in a new commit